src/components/dataset/property/new.vue

```diff
@@ -93,6 +93,7 @@ const submit = () => {
 
 <i18n lang="json5">
 {
+  // @transifexKey component.DatasetOverviewNewProperty
```

Comment:
Mostly filenames don't really matter, but the one place where they do have an effect is in Transifex. For Transifex, the string key is based on the filename, under the assumption that the filename will match the component name. (I couldn't think of a better way for our Transifex scripts to figure out the component name.) We don't want these messages to be retranslated, which is why I'm specifying `@transifexKey` here.